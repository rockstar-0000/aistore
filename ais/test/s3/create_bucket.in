s3cmd mb s3://$BUCKET --host=$HOST $PARAMS --region us-west-1 --host-bucket="$HOST/%(bucket)"
s3cmd ls $PARAMS --host=$HOST --region us-west-1 --host-bucket="$HOST/%(bucket)" | grep "$BUCKET"
s3cmd ls s3://$BUCKET --host=$HOST $PARAMS --region us-west-1 --host-bucket="$HOST/%(bucket)"
s3cmd rb s3://$BUCKET --host=$HOST $PARAMS --region us-west-1 --host-bucket="$HOST/%(bucket)"
