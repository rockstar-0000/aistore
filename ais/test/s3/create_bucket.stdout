Bucket 's3://$BUCKET/' created
^(.*) $BUCKET$
Bucket 's3://$BUCKET/' removed
